packages/presentation/lib/screen/splash/splash_screen.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+import '../../navigation/base_page.dart';
+
+class SplashScreen extends StatelessWidget {
+  const SplashScreen({Key? key}) : super(key: key);
+  static const gradientTopColor = Color(0xFFE51937);
+  static const gradientBottomColor = Color(0xFFDB5252);
```

Comment:
need to think if we can move these values to a separate file. maybe then can be used on another screen

Reply:
done